OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -808,9 +747,7 @@ void FindNewUnits(Actor self)
 
 			foreach (var a in newUnits)
 			{
-				if (a.Info.HasTraitInfo<HarvesterInfo>())
-					QueueOrder(new Order("Harvest", a, false));
```

Comment:
This change isn't resembled in the new manager. Is it save to drop this order?

Reply:
It should be. I can't imagine any situation in which the HarvManager won't catch and give an order to an idle harvester.
And as long as there are resources in the vincinity of the production facility that spawned it, the harvesters' own FindResources will trigger.

So as far as I can tell, this Order was redundant.

Reply:
I've seen it happenning that since AI new units reach the rally point with an attack-move sometimes the AI used new war miners in base defence due to the attack move. But these units went to harvest after they defeated the threat in the base and started idling thenafter, so I presume this'll be fine as well.